fcli-core/fcli-common/src/main/java/com/fortify/cli/common/output/cli/mixin/OutputOptionsArgGroup.java

```diff
@@ -28,13 +28,13 @@
 import picocli.CommandLine.Option;
 
 public final class OutputOptionsArgGroup implements IOutputOptions {
-    @Option(names = {"-o", "--output"}, order=1, converter = OutputFormatConfigConverter.class, completionCandidates = OutputFormatIterable.class, paramLabel = "type[=<args>]")
+    @Option(names = {"-o", "--output"}, order=1, converter = OutputFormatConfigConverter.class, completionCandidates = OutputFormatIterable.class, paramLabel = "<type+args>", defaultValue = "table")
```

Comment:
Adding a default value here is wrong, as default output format depends on the command being run. Please remove, and update resource bundle description accordingly.